tools/gluten-it/common/pom.xml

```diff
@@ -186,6 +193,40 @@
           </execution>
         </executions>
       </plugin>
+
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-shade-plugin</artifactId>
+        <version>3.4.1</version>
+        <configuration>
+          <shadedArtifactAttached>false</shadedArtifactAttached>
+          <artifactSet>
+            <includes>
+              <include>io.trino.tpcds:tpcds</include>
+              <include>io.trino.tpch:tpch</include>
+              <include>com.google.guava:guava</include>
+              <include>io.glutenproject:gluten-it-common</include>
```

Comment:
I'm not sure how it works before. IIUC, Trino tpcds/tpch depend on high version guava which can be conflict with Spark guava... This pr shade them to make sure it's right.

Reply:
@zhztheplayer could you help to take a look here?

Reply:
will take a look asap